test/tsconfig.json

```diff
@@ -5,6 +5,8 @@
       "outDir": "../dist/test",
       "target": "es2017",
       "module": "commonjs",
+      "experimentalDecorators": true,
```

Comment:
This discussion may have happened already, but since we're adding these to most of the tsconfigs is it worth just having these added to the base config instead?

Reply:
This was on going discussion.  I think it would make sense to add it to the base config, considering injection cant work without it.

Reply:
Made a PR for `experimentalDecorators` addition to the base config: https://github.com/HadoukenIO/service-tooling/pull/56

Note that this would require a service-tooling deployment, so this change in this PR should remain until that is done.

Reply:
Fine by me!